microsoft-edge/webview2/concepts/process-related-events.md

```diff
@@ -0,0 +1,190 @@
+    - COM: [COREWEBVIEW2_PROCESS_FAILED_REASON](<link>)
+    - .NET: [CoreWebView2ProcessFailedReason](<link>)
+    - WinRT: [CoreWebView2ProcessFailedReason](<link>)
+
+    > [!NOTE]
+    > For unresponsive renderer _failure kind_, the _failure reason_ is always `unresponsive`.
+
+Additionally, the `ProcessFailed` event provides more detailed information about the reported failure, including:
+  * Exit code
+  * Process description (utility only)
+  * Frames information (renderer only)
+
+Your application can leverage this information for diagnostics and other scenarios. For more information about when these details are provided and how your application can use them, see:
+  - COM: [ICoreWebView2ProcessFailedEventArgs](<link>), [ICoreWebView2ProcessFailedEventArgs2](<link>)
+  - .NET: [CoreWebView2ProcessFailedEventArgs](<link>)
+  - WinRT: [CoreWebView2ProcessFailedEventArgs](<link>)
+
+### Unexpected Exits
+The processes in the _WebView2 Process Group_ can be associated to one or many WebView2 controls in your application. For example:
+
+  * **Main browser process.** There is a single main browser process in the _WebView2 Process Group_. Every WebView2 control with the same environment configuration will share this process.
+  * **Renderer process.** Renderer processes can be associated many-to-many with the WebView2 controls in your application. The details about how this works depend on many factors, including what sites are loaded in the WebView2 control (main frame and subframes), system resources, and runtime configuration. A single renderer process in the _WebView2 Process Group_ can be associated with one or many WebView2 controls in your app.
+  * **GPU process.** There is a single GPU process in the _WebView2 Process Group_. It is associated with all WebView2 controls using this _WebView2 Process Group_.
+  * **Utility processes.** Utility processes host one or more _services_ in the _WebView2 Process Group_. Each utility process supports the entire _WebView2 Process Group_ and is thus associated with all WebView2 controls using this _WebView2 Process Group_.
+
+  > [!NOTE]
+  > Do not rely on the details about how processes are associated to each WebView2 control, as they are part of the evolving Chromium architecture and subject to change even due to configuration and system conditions. For more information see [Process Model](process-model.md).
+
+When a process in the _WebView2 Process Group_ exits unexpectedly (for example, due to a crash), every WebView2 control **associated** to it will raise a `ProcessFailed` event. You can use the [Process Failure Details](<link>) to decide how to handle each case. The following are some examples of what these cases can be:
```

Comment:
```suggestion
When a process in the _WebView2 Process Group_ exits unexpectedly (for example, due to a crash), every WebView2 control **associated** to it will raise a `ProcessFailed` event. You can use the [Process failure details](#process-failure-details) (above) to decide how to handle each case. The following are some examples of what these cases can be:
```